src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,108 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import java.io.IOException;
+
+public class Storage {
+
+    private String pathName;
+    private String fileName;
+
+    public Storage(String pathName, String fileName) {
+        this.pathName = pathName;
+        this.fileName = fileName;
+    }
+
+    /**
+     * Returns a TaskList that stores tasks retrieved from the data/duke.txt if the file exits
+     *
+     * @return a TaskList with tasks stored in the txt file.
+     */
+    public TaskList load() {
```

Comment:
Perhaps a couple of line breaks in this function will improve readability?

Reply:
I tried line break but 2103 style check alerts unnecessary line break tho.
Line matches the illegal pattern 'Trailing whitespace'. (16:0) [Regexp]